src/dom.js

```diff
@@ -111,6 +111,9 @@ export function closest(element, callback) {
  * @return {?Element}
  */
 export function closestByTag(element, tagName) {
+  if (element.closest) {
```

Comment:
I changed a test slightly: the previous version was a bit biased because of likely query selector result caching. Still, `Element.closest` was about 20-25% faster.